README.md

```diff
@@ -155,18 +156,22 @@ from **server/cmd/dcrdex**.
 
 ## Client Installation
 
-The client is in early development, and is not fully functional. Instructions
-are listed for development purposes.
+The client is still immature and changing rapidly, but it is fully functional.
+Please ensure you are building from a current snapshot of the master branch.
```

Comment:
I would not say it's fully functional until #513 is in. Until that point, there is still a path to losing funds.